lib/js/atoms/SurveyQuestion/SurveyQuestionScale.vue

```diff
@@ -0,0 +1,240 @@
+<template>
+	<div class="surveyQuestionScale">
+		<ds-modal v-if="showModal" @close-modal="showModal = false">
+			<slot name="explanation" />
+			<div slot="footer">
+				<ds-button :type="BUTTON_TYPES.OUTLINED" @click.native="showModal = false">
+					OK, rozumiem
+				</ds-button>
+			</div>
+		</ds-modal>
+		<ds-card>
+			<template slot="content">
+				<div class="surveyQuestionScale__title">
+					<span>{{ title }}</span>
+					<icon-button
+						v-if="$slots.explanation"
+						class="surveyQuestionScale__explanation"
+						:color="BUTTON_COLORS.MINOR_SUPPORTING"
+						:icon="ICONS.FA_QUESTION_CIRCLE"
+						:size="ICON_SIZES.MEDIUM"
+						:touchable="false"
+						@click.native="showModal = true"
+					/>
+				</div>
+				<div class="surveyQuestionScale__content">
+					<template v-for="option in scaleOptions">
+						<div
+							v-if="option.standAlone"
```

Comment:
1. This is a hidden option that you need to find here 😞  Can we export an option interface to make it easier?
2. It should be `standalone` https://writingexplained.org/standalone-or-stand-alone

Reply:
[`97af32b` (#71)](https://github.com/bethinkpl/design-system/pull/71/commits/97af32b3fdc55953bdc020f1b452474f96e9f10c)